prototypes/process-explorer/dotnet/src/MorganStanley.ComposeUI.ProcessExplorer.Core/ProcessInfoAggregator.cs

```diff
@@ -55,10 +52,22 @@ internal class ProcessInfoAggregator : IProcessInfoAggregator
         _processInfoMonitor = processInfoMonitor;
         _subsystemController = subsystemController;
 
-        _processInfoMonitor.ProcessIds
+        SetProcessIdsNotificationMethods();
+    }
+
+    private void SetProcessIdsNotificationMethods()
+    {
+        try
+        {
+            _processInfoMonitor.ProcessIds
             .Select(kvp => Observable.FromAsync(async () => await PushNotification(kvp)))
             .Concat()
             .Subscribe();
+        }
+        catch (ArgumentNullException exception)
```

Comment:
This looks suspicious: when can this code throw an ArgumentNullException? Wouldn't it be better not to call it with null argument(s)?

Reply:
My purpose was to put it in a try catch block as when I was creating the tests and I mocked the processInfoMonitor due it is not instantitated just mocked it caused an error. I'll update the PR. :)